rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf

```diff
@@ -87,54 +87,40 @@ SecRule REQUEST_LINE "!@rx (?i)^(?:(?:[a-z]{3,10}\s+(?:\w{3,7}?://[\w\-\./]*(?::
 #
 # -=[ Targets, characters and html entities ]=-
 #
-# 920120: PL1 : FILES_NAMES, FILES
-#    ['\";=] or any of the following entities:
-#    &[aeiouclnrszg]acute;
-#    &[cdelnrstz]caron;
-#    &[cgklnrst]cedil;
-#    &[aeioucghjswy]circ;
-#    &[aeiou]grave;
-#    &[au]ring;
-#    &[anoi]tilde;
-#    &[aeiouy]uml;
-#    &amp;
-#    &apos;
-#    &nbsp;
-#    &oslash;
+# 920120 + 920122: PL1 : FILES_NAMES, FILES
+#    ['\";=] or any frequently used HTML entity (see 920120.data).
 #
 # 920121: PL2 : FILES_NAMES, FILES
-#                ['\";=] : ' " ; = meta-characters
+#    ['\";=] : ' " ; = meta-characters
 #
 # -=[ References ]=-
 # https://www.owasp.org/index.php/ModSecurity_CRS_RuleID-96000
 # http://www.ietf.org/rfc/rfc2183.txt
 #
-# The regular expression in the active rule 920120 demands a PCRE-compatible
-# regular expression engine. If you are using a non-PCRE engine, you can generate
-# an equivalent regular expression by generating the expression (the resulting
-# expression will not use negative lookbehind):
-#   cd util/regexp-assemble
-#   ./regexp-assemble.py 920120-no-backtracking
-#
-# This alternative regex is not the default one, since it comes with a severe
-# performance impact, namely for larger files.
-#
-# Caution: The performance impact of the alternative regex can lead to
-# a DoS for larger files.
-#
-# Please see https://coreruleset.org/20210106/introducing-msc_retest/ for
-# a thorough discussion and detailed performance data.
-#
-# The regex in the following enabled rule is not supported by non-PCRE
-# regular expression engines (?<!re).
-#
 # To rebuild the regular expression:
 #   cd util/regexp-assemble
 #   ./regexp-assemble.py 920120
```

Comment:
Probably this needs to be `./regexp-assemble.py data/920120.data`?

Reply:
No, actually. The script knows where to look and how to find the file for the rule number ;)